src/lib/features/playground/feature-evaluator/strategy/strategy.ts

```diff
@@ -73,6 +73,7 @@ export class Strategy {
                     ...constraint,
                     value: constraint?.value?.toString() ?? undefined,
                     result: this.checkConstraint(constraint, context),
+                    caseInsensitive: constraint.caseInsensitive ?? false,
```

Comment:
constraint.caseInsensitive is of type `boolean | null | undefined` while caseInsensitive is `boolean`